app/views/layouts/_cookies_consent.html.erb

```diff
@@ -0,0 +1,40 @@
+<% unless cookies[:cookies_policy]%>
+  <div class="app-cookie-banner" style="background-color: #f3f2f1;">
+    <div class="app-cookie-banner__message" id="cookies_consent_mechanism">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-m">Cookies on HMRC services</h1>
+          <p>We use some essential cookies to make our services work.</p>
+          <p>We would like to set additional cookies so we can remember your settings,
+            understand how people use our services and make improvements.
+          </p>
+          <%= button_to  "Accept additional cookies", cookies_consent_accept_path, class: 'govuk-button
+               cookie_accept_all',
+                         action: 'accept_cookies' %>
+          <%= button_to  "Reject additional cookies", cookies_consent_reject_path, class: 'govuk-button
```

Comment:
Since you mention the URL to post to: cookies_consent_reject_path do you need to specify the action?